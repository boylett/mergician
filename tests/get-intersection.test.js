const getIntersection = require('../src/get-intersection');

const arr1 = ['a', 'b', 'c'];
const arr2 = ['b', 'c', 'd'];
const arr3 = ['c', 'd', 'e'];

test('gets intersection of two arrays', () => {
    const result = getIntersection(arr1, arr2);

    expect(result).toMatchSnapshot();
});

test('gets intersection of more than two arrays', () => {
    const result = getIntersection(arr1, arr2, arr3);

    expect(result).toMatchSnapshot();
});
