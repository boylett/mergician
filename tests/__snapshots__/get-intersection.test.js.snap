// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`gets intersection of more than two arrays 1`] = `
Array [
  "c",
]
`;

exports[`gets intersection of two arrays 1`] = `
Array [
  "b",
  "c",
]
`;
